src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -81,14 +81,14 @@ static auto logger = ert::get_logger("job_queue.lsf_driver");
   to source schell specific input files prior to executing your
   job. This can be achieved with the LSF_LOGIN_SHELL option:
 
-     lsf_driver_set_option( driver , LSF_LOGIN_SHELL , "/bin/csh" );
```

Comment:
About what?

Reply:
Since  @berland has done a lot of work to make the drivers work reliably on azure, he might have something to say about this. Otherwise it seems fine.

Reply:
I think onprem RHEL nodes and Azure RHEL nodes should be identical in this regard, so this looks fine to me 🚀